TicketAir.py

```diff
@@ -0,0 +1,296 @@
+        self.name = name
+        self.tickets = tickets
+        self.address = address
+        self.password = password
+        self.login = login
+
+    def addFlight(self, flight):
+        flight.add(self)
+
+    def addPlane(self, plane):
+        plane.add(self)
+
+    def cancelFlight(self, flight):
+        flight.cancel(self)
+
+    def removePlane(self, plane):
+        plane.remove(self)
+
+
+class DataBase:
+    def create_tables(self):
+        con = sqlite3.connect('ticketair.db')
+        cur = con.cursor()
+        try:
+            cur.execute('''CREATE TABLE flights
+                            (origin text, destination text, flightNumber text, planeTailNumber text, price real, time real, capacity integer, sold integer)
+                        ''')
+        except sqlite3.OperationalError:
+            pass
+        try:
+            cur.execute('''CREATE TABLE planes
+                            (tailNumber text, year integer, flightMiles real, capacity integer)
+                        ''')
+        except sqlite3.OperationalError:
+            pass
+        try:
+            cur.execute('''CREATE TABLE users
+                            (userID text, name text, ticketsIDs text, isAdmin integer, password text, login text)
+                        ''')
+        except sqlite3.OperationalError:
+            pass
+        try:
+            cur.execute('''CREATE TABLE tickets
+                            (flightNumber text, price real, places integer, luggageID integer)
+                        ''')
+        except sqlite3.OperationalError:
+            pass
+        con.commit()
+        con.close()
+
+    def addFlight(self, flight):
+        con = sqlite3.connect('ticketair.db')
+        cur = con.cursor()
+        cur.execute('INSERT INTO flights VALUES ' + flight.values())
+        con.commit()
+        con.close()
+
+    def addPlane(self, plane):
+        con = sqlite3.connect('ticketair.db')
+        cur = con.cursor()
+        cur.execute('INSERT INTO planes VALUES ' + plane.values())
+        con.commit()
+        con.close()
+
+    def addUser(self, user):
+        con = sqlite3.connect('ticketair.db')
+        cur = con.cursor()
+        cur.execute('INSERT INTO users VALUES ' + user.values())
+        con.commit()
+        con.close()
+
+    def getPlane(self, tailNumber):
+        con = sqlite3.connect('ticketair.db')
+        cur = con.cursor()
+        cur.execute(f'SELECT * FROM planes WHERE tailNumber = "{tailNumber}"')
+        tailNumber, year, flightMiles, capacity = cur.fetchone()
+        con.close()
+        return Plane(year, flightMiles, capacity, tailNumber)
+
+    def getDestinations(self, origin=None):
+        con = sqlite3.connect('ticketair.db')
+        cur = con.cursor()
+        if origin == None or origin == '':
+            cur.execute(f'SELECT DISTINCT destination FROM flights')
+        else:
+            cur.execute(f'SELECT DISTINCT destination FROM flights WHERE origin="{origin}"')
```

Comment:
If you need to consider SQL injection attacks in this project, then this is one of places it can be exploited.